Snippets/Core/Core_7/UpgradeGuides/6to7/Scheduling/ExecuteScheduledTaskUsingTaskRun.cs

```diff
@@ -0,0 +1,47 @@
+﻿namespace Core7.UpgradeGuides._6to7.Scheduling
+{
+    using NServiceBus;
+    using NServiceBus.Logging;
+    using System;
+    using System.Threading.Tasks;
+
+    class ExecuteScheduledTaskUsingTaskRun
+    {
+        async Task TaskRun(IEndpointInstance endpointInstance, ILog logger)
+        {
+            #region 6to7ExecuteScheduledTaskUsingTaskRun
+            await endpointInstance.ScheduleEvery(
+                    timeSpan: TimeSpan.FromMinutes(5),
+                    task: pipelineContext =>
+                    {
+                        Task.Run(() =>
```

Comment:
I would strongly discourage using the continuation style. 

It is much better to wrap it in a method or local function, something like


```
_ = DoWork();

static async Task DoWork() {
  try {
     // Do some work here
     logger.Error($"Failed to execute MyTask'", ex);
  } catch (Exception ex) {
     logger.InfoFormat("MyTask taskcompleted");
  }
}
```